src/renderer/mission/Mission.js

```diff
@@ -56,34 +50,64 @@ export default class MissionContainer extends Component {
         },
       ],
     };
-  }
 
-  render() {
-    const { missions } = this.state;
+    width = {
+      description: 0.6,
+      status: 0.4,
+    };
+
+    heightCache = new CellMeasurerCache({
+      fixedWidth: true,
+      minHeight: 40,
+    });
+
+    _rowGetter = ({ index }) => this.state.missions[index];
```

Comment:
what is the convention for using _ in method names?

Reply:
I see people doing _name when putting the function into a parameter.

```js
<Table
  width={width}
  height={height}
  headerHeight={40}
  rowHeight={this.heightCache.rowHeight}
  rowCount={this.state.missions.length}
  rowGetter={this._rowGetter}
  onRowClick={this._onRowClick}
>
```

Seems to be a standard of some sort when it comes to naming functions. Consider them to be like private functions (people call them callbacks)

Reply:
Resolved in https://github.com/NGCP/GCS/commit/7281b09d05d28cd92286e87185713d2b03d56da1